package.json

```diff
@@ -22,37 +22,23 @@
     "@angular/platform-browser": "~12.2.0",
     "@angular/platform-browser-dynamic": "~12.2.0",
     "@angular/router": "~12.2.0",
-    "@blackbaud/auth-client": "^2.48.0",
-    "@blackbaud/skyux-lib-clipboard": "^4.0.0",
-    "@blackbaud/skyux-lib-code-block": "^4.0.2",
-    "@blackbaud/skyux-lib-media": "^4.0.0",
-    "@blackbaud/skyux-lib-restricted-view": "^4.2.1",
-    "@blackbaud/skyux-lib-stache": "^4.2.2",
-    "@skyux/ag-grid": "^5.0.0-beta.1",
     "@skyux/animations": "^5.0.0-beta.1",
     "@skyux/assets": "^5.0.0-beta.0",
     "@skyux/autonumeric": "^5.0.0-beta.1",
     "@skyux/config": "^5.0.0-beta.1",
     "@skyux/core": "^5.0.0-beta.9",
-    "@skyux/data-manager": "^5.0.0-beta.2",
```

Comment:
No that you've removed `@skyux/data-manager`, you'll need to change the import paths in your documentation folder to `project/data-manager/src/public-api`.